src/main/resources/messages_en.properties

```diff
@@ -583,6 +583,13 @@ germplasm.name.preferred.required=Preferred name field is required
 gids.invalid=The following gid(s): {0} were not found in the system
 lot.Ids.invalid=The following lotId(s): {0} were not found in the system
 
+germplasm.name.type.code.invalid=The code {0} already exists.
+germplasm.name.type.name.invalid=The name {0} already exists.
+germplasm.name.type.code.field.invalid=The code field should not be empty.
```

Comment:
@cuenyad we may need to modify this messages. I would prefer to remove the word field because the service should be abstract from where it is being called.